### ISSUE TITLE
Allow use of :terminal and :Dispatch for aync compiling

### ISSUE THREAD

**Issue by gauteh** [vim-latex/vim-latex#101]
Hi,

as far as I can see it is not possible to customize the compilation command (for PDF in my case) so that it wraps the compilation through :terminal or :Dispatch. This would be especially useful for documents that require a long time to compile.

**Comment by gerw** [comment]
Vim-latex uses just `make[!]` for compilation, see `Tex_CompileLatex()` in `ftplugin/latex-suite/compiler.vim`.

However, for compilation of larger projects I would recommend to use an external program, e.g., `latexmk`. Afterwards, you can display the errors with `:cg | :copen`.

**Comment by gauteh** [comment]
Oh, I thought I would miss out on multiple compiles (if necessary) if I 
just used make.

-gaute

**Comment by gerw** [comment]
AFAIK, `:make` does just one compilation. However, `Tex_RunLaTeX()` takes care of multiple compiles.

**Comment by gauteh** [comment]
Gerd Wachsmuth writes on september 4, 2017 8:53:
> AFAIK, `:make` does just one compilation. However, `Tex_RunLaTeX()` takes care of multiple compiles.

Yeah - that's the one that would be great to run in the background. 
Either through dispatch or Vim Jobs.

**Comment by gerw** [comment]
Yes, this would be a possibility. However, I do not have the time to implement it. Feel free to issue a pull request.

**Comment by gauteh** [comment]
Gerd Wachsmuth writes on september 4, 2017 9:14:
> Yes, this would be a possibility. However, I do not have the time to implement it. Feel free to issue a pull request.

Alright, I think I will go for latexmk like you suggested for projects 
where this is an issue. Thanks for the help. I don't have time at the 
moment either.
